nanosynth.rb

```diff
@@ -28,11 +28,13 @@
 ###
 ### http://www.joelstrait.com/nanosynth_create_sound_with_ruby/
 
-gem 'wavefile', '=0.7.0'
+gem 'wavefile'
 require 'wavefile'
 
 OUTPUT_FILENAME = "mysound.wav"
 SAMPLE_RATE = 44100
+FORMAT_BITS_PER_SAMPLE = "pcm_16" # Supported values are pcm_8, pcm_16, pcm_24, pcm_32, float_32, and float_64
```

Comment:
The WaveFile gem was a champ and I didn't even consider that having a symbol was ideal.
Could change the definition to do `.to_sym` in the definition, or `WaveFile::Format.new(:mono, FORMAT_BITS_PER_SAMPLE.to_sym, SAMPLE_RATE))`, or of course simply update to `:pcm_16` and `:pcm_*` in the comment.
I'd be happy to do a PR for those changes.